src/Coupon/SyncerHooks.php

```diff
@@ -322,4 +353,22 @@ protected function set_already_scheduled_to_update( int $coupon_id ): void {
 	protected function set_already_scheduled_to_delete( int $coupon_id ): void {
 		$this->set_already_scheduled( $coupon_id, self::SCHEDULE_TYPE_DELETE );
 	}
+
+	/**
+	 * Schedules notifications for an updated product
```

Comment:
I'm assuming this was copied over from `Product/SyncerHooks.php` but we're referencing a product here:

```suggestion
	 * Schedules notifications for an updated coupon
```